docs/src/rsh/compute/index.md

```diff
@@ -1840,6 +1840,11 @@ lastConsensusSecs()
 
 @{defn("lastConsensusSecs")} is like `{!rsh} lastConsensusTime`, except it returns the network seconds.
 
+```reach
+load: /examples/realtime/index.rsh
+range: 18 - 18
```

Comment:
Same thing here. Samples should provide more context than the syntax reference. 

Let's find a different example so we're not showing a duplicate sample right after the other.